server_src/lib/logger.js

```diff
@@ -1,32 +1,41 @@
 const winston = require("winston");
 
+const isProduction = process.env.NODE_ENV === "production";
+
+const dtFormat = new Intl.DateTimeFormat('en-GB', {
```

Comment:
Cool formatter I found! Didnt know it existed by default myself. Much shorter time stamp.

@NotExpectedYet do note: adding the timestamps causes two timestamps to be printed in the log files. What's up with that?